services/spring-boot-starter-datawave/src/main/resources/static/css/screen.css

```diff
@@ -63,6 +63,7 @@ td {
 	padding-bottom: 0.15em;
 	color: #333333;
     font-size: 10pt;
+    word-wrap:break-all
```

Comment:
I wonder if we want to have character-level wrapping for every cell in every table. It might be better to limit this to a td with a class assigned, and then where the behavior is desired, we add the class on to the column.

Reply:
BTW: Jeff is currently this as requested Brian.....